packages/core/src/config/load.ts

```diff
@@ -120,25 +135,106 @@ export function getLintRawConfigWithMergedContentByPriority(
   };
 }
 
-async function resolveExtends(lintConfig: LintRawConfig): Promise<LintRawConfig> {
-  if (!lintConfig.extends || !lintConfig.extends.length) return lintConfig;
+export function resolveNestedPlugins(
+  configPath: string,
+  pluginConfigPath: string,
+  plugin: string | Plugin,
+): string | Plugin | undefined {
+  if (!plugin) {
+    return;
+  }
+
+  if (!isString(plugin)) {
+    return plugin;
+  }
+
+  if (isAbsoluteUrl(plugin as string)) {
+    throw new Error(
+      `Error configuration format not detected (${plugin}). Nested plugin should be path to local file`,
+    );
+  }
+  const pluginPath = path.resolve(path.dirname(pluginConfigPath), plugin as string);
+
+  return path.resolve(path.dirname(configPath), pluginPath);
+}
+
+async function resolveApis(
+  apis: Record<string, Api>,
+  configPath: string = '',
+  resolve?: RawResolveConfig,
```

Comment:
It is usually better to use object argument notation when there's no clear precedence to multiple arguments.